Makefile

```diff
@@ -647,12 +647,22 @@ $(BUILD_DIR)/profiles.ttl: $(BUILD_DIR)/evolutionary-profiles.ttl $(BUILD_DIR)/g
 # 2. Closures = subclass + instance)
 # 3. taxa-sim = taxa-ics + taxa-expect-scores + taxa-pairwise-sim
 # 4. gene-sim = gene-ics + gene-expect-scores + gene-pairwise-sim
+# 5. build-time
+
+# Insert KB build time in DB
+$(BUILD_DIR)/build-time.ttl: $(SPARQL)/build-time.sparql $(BUILD_DIR)/phenoscape-kb.ttl
+	$(ARQ) \
+	--data=$(BUILD_DIR)/phenoscape-kb.ttl \
+	--results=NTRIPLES \
```

Comment:
How can you output ntriples from a `SELECT`? This doesn't need to depend on `phenoscape-kb.ttl`.